.gitignore

```diff
@@ -32,3 +32,7 @@ build-tests/
 packages/
 .DS_Store
 *.bootstrap.cache
+
+# Direnv
```

Comment:
Who creates those files?

Reply:
[direnv](https://direnv.net/). It works well with NixOS because it reads the `default.nix` file whenever I cd into a directory and sets up the environment variables accordingly.